op-batcher/Makefile

```diff
@@ -24,8 +24,14 @@ ifeq ($(shell uname),Darwin)
 	FUZZLDFLAGS := -ldflags=-extldflags=-Wl,-ld_classic
 endif
 
+# Check if TARGETARCH is set to arm64 and adjust compilers accordingly
+ifeq ($(TARGETARCH),arm64)
+    CC := aarch64-linux-gnu-gcc
+    CXX := aarch64-linux-gnu-g++
+endif
```

Comment:
It would be nice to have it check amd64 and set to amd, then if-else-branch on arm64 check, and final else branch that sets CGO_ENABLED=0.

This way I am able to build an amd64 image from my arm64 machine. And we'll have a default that makes sense if people try to target other architectures.

CGO_ENABLED=0 should entirely avoid the C toolchain, and use pure-go mode (no native platform optimizations, e.g. slower signature verification fallback). We tried to just use that as default, but for some reason it was broken (probably a dependency which didn't/doesn't support pure-go compilation).

Same applies to other make files.

Reply:
Decided to use CGO_ENABLED=0 to allow cross compile